doc/release_notes/release_notes_2.1.rst

```diff
@@ -67,9 +67,40 @@ including these:
 
 Fixed Issues Details
 ********************
-
-.. comment Issues should be carefully formatted like this
-   - :acrn-issue:`3715` -  Add support for multiple RDT resource allocation and fix L3 CAT config overwrite by L2
+- :acrn-issue: `4047` -  [WHL][Function][WaaG] passthru usb, Windows will hang when reboot it
```

Comment:
There must not be a space after the colon in ``:acrn-issue:`4047` ``

Reply:
Will take care this issue next time.
Thanks.